doc/data/messages/i/import-error/good.py

```diff
@@ -1 +1,2 @@
-# This is a placeholder for correct code for this message.
+# package installed in your venv, globally, user space
+import third_party
```

Comment:
```suggestion
import sphinx
```
Trying to be clever by using a package we have in our env, but realistically this is not going to be something we can demonstrate by using our doc framework. If this doesn't work could you remove import-error from this merge request ?